app/src/pages/AbstractStreamEditPage/RelatedProjects.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { MarketplaceProductTile } from '$mp/components/Projects'
+import { TABLET, PHONE } from '$shared/utils/styled'
+import { TheGraphProject, getProjects } from '$app/src/services/projects'
+import useIsMounted from '$shared/hooks/useIsMounted'
+
+const Container = styled.div`
+    margin-top: 80px;
+    margin-bottom: 120px;
+`
+
+const Header = styled.div`
+    font-size: 34px;
+    line-height: 34px;
+    color: #000000;
+    padding-bottom: 40px;
+`
+
+const ProjectGrid = styled.div`
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: 36px;
+
+    @media (${PHONE}) {
+        grid-template-columns: 1fr 1fr;
+    }
+
+    @media (${TABLET}) {
+        grid-template-columns: 1fr 1fr 1fr 1fr;
+    }
+`
+
+type Props = {
+    streamId: string,
+}
+
+const RelatedProjects: React.FC<Props> = ({ streamId }) => {
+    const isMounted = useIsMounted()
+    const [projects, setProjects] = useState<TheGraphProject[]>([])
+
+    useEffect(() => {
+        const loadProjects = async () => {
+            const result = await getProjects(null, 4, 0, null, streamId)
+            if (isMounted()) {
+                setProjects(result.projects)
+            }
+        }
+        loadProjects()
+    }, [isMounted, streamId])
```

Comment:
I'd rather avoid `useIsMounted` in this case. In MANY cases, actually. Think of what you've got here:

1. Mount the component for stream "A".
2. Rerender it (w/o unmounting) for stream "B".

If, for whatever reason, `getProjects` for "A" finishes AFTER the one for "B", then for stream "B" you'll see projects related to "A".

```suggestion
    useEffect(() => {
        let mounted = true

        const loadProjects = async () => {
            const result = await getProjects(null, 4, 0, null, streamId)

            if (mounted) {
                setProjects(result.projects)
            }
        }

        loadProjects()
        
        return () => {
            mounted = false
        }
    }, [streamId])
```